packages/rke2-calico/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://github.com/projectcalico/calico/releases/download/v3.23.1/tigera-operator-v3.23.1.tgz
-packageVersion: 04
+url: https://github.com/projectcalico/calico/releases/download/v3.23.3/tigera-operator-v3.23.3.tgz
+packageVersion: 05
```

Comment:
When the tarball changes, we move the packaVersion back to 01

Reply:
Thanks for your comment.